### PR TITLE
<FormField/> - fix label placement

### PR DESCRIPTION
**Pull request by daugerdas**
### 🔦 Summary

Fix label placement on `<FormField/>` to not wrap elements when `labelPlacement` prop is set to left or right

Relevant GitHub issue https://github.com/wix/wix-style-react/issues/6356 (labelPlacement="left" style is broken with long text)
Description: [DSM-1202](https://jira.wixpress.com/browse/DSM-1202)

[Steps to reproduce this issue](https://www.wix-style-react.com/?path=/story/introduction-playground--playground&snippet=d0255664-0296-4e2f-bc53-a8dc2e4b0b6f):
```jsx
<div style={{ width: 200 }}>
  <FormField label="This line is not aligned to left" labelPlacement="left">
    <ToggleSwitch size="medium" />
  </FormField>
</div>
```

Currently/Updated:
image image

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6392

File: src/FormField/FormField.st.css [hunk 1]
```diff
@@ -24,11 +24,11 @@
 }
 
 .root:labelPlacement(right) {
-  flex-direction: row;
+  flex-flow: row nowrap;
```
Comment (mykas) [inline src/FormField/FormField.st.css:27]: Nice solution! I really like it!

**Review by mykas — APPROVED** [review]
Good job! cleanest solution ever

File: src/FormField/FormField.st.css (hunk 1 above)
Comment (daugerdas) [reply src/FormField/FormField.st.css:27]: Thank you!

**Pull request merged**